src/Data/LLVM/BitCode/IR/Metadata.hs

```diff
@@ -627,20 +630,32 @@ parseMetadataEntry vt mt pm (fromEntry -> Just r) =
         <*> (mdForwardRefOrNull ctx mt <$> parseField r 11 numeric) -- dicuSubprograms
         <*> (mdForwardRefOrNull ctx mt <$> parseField r 12 numeric) -- dicuGlobals
         <*> (mdForwardRefOrNull ctx mt <$> parseField r 13 numeric) -- dicuImports
-        <*> if recordSize <= 15
-            then pure Nothing
-            else mdForwardRefOrNull ctx mt <$> parseField r 15 numeric -- dicuMacros
-      dicuDWOId <-
-        if recordSize <= 14
-        then pure 0
-        else parseField r 14 numeric
-      dicuSplitDebugInlining <-
-        if recordSize <= 16
-        then pure True
-        else parseField r 16 nonzero
-      let dicu' = dicu dicuDWOId dicuSplitDebugInlining
+        <*> (if recordSize <= 15
```

Comment:
Is there a reason that the 15 and 14 size cases don't appear in numerical order?

Reply:
Not a good one. It was just the minimal change: that's the order they were in before. But, since I'm changing all those lines anyway, I might as well put them in numerical order.

Reply:
Oh, actually, there is a better reason: that's the order they were in before so that's the order they appear in the datatype in `llvm-pretty`. If we change that, we'd have to change all clients of the datatype.

Reply:
Gotcha. Thanks!